js/index.js

```diff
@@ -7,51 +10,57 @@ const saveToLocalStorage = () => {
 const getDataFromLocalStorage = () => {
   const data = JSON.parse(localStorage.getItem('Library'));
   if (data !== null) {
-    listBooks = data;
+    listBooks.setBooks(data.books);
+  } else {
+    saveToLocalStorage();
   }
 };
 
 function displayBooks() {
   const section = document.querySelector('#book-list');
-  let books = '';
+  let books = '<table>';
   getDataFromLocalStorage();
-
-  listBooks.forEach((book, index) => {
-    books += `<article class="book">
-    <p>${book.title} <br> ${book.author}</p>
-    <button type="button" id="${index}" class="remove-btn" onclick="removeBook(${index})">Remove</button>
-    <hr>
-  </article>`;
+  listBooks.books.forEach((book, index) => {
+    books += `<tr>
+    <td>
+      <article class="book">
+        <p>"${book.title}" by ${book.author}</p>
+        <button type="button" id="${index}" class="btn remove-btn" onclick="removeBook(${index})">Remove</button>
+      </article>
+    </td>
+  </tr>
+  `;
   });
 
-  if (listBooks.length === 0) {
-    books = '<p>Library is empty...</p>';
+  if (listBooks.books.length === 0) {
+    books += '<tr><td<p class="empty-libray">Library is empty...</p></td></tr>';
   }
+  books += '</table>';
   section.innerHTML = books;
 }
 
 function removeBook(id) {
```

Comment:
- [ ] I couldn't see that you refactor your methods to be in a class I higly recommend to refactor it I saw your book.js but in the requirements you should refactor the methods that already exist not to create a methods to just push but you should put the method that you already created in class and It is not working the adding functionality now I recommend to fix the problem to be able to add a new book  ✅♻💯

https://user-images.githubusercontent.com/58553711/211687714-b4e4cca2-1594-4928-a982-fe332f41d0e4.mp4